src/app/error-summary/error-summary.component.html

```diff
@@ -0,0 +1,10 @@
+<div #errorSummary tabindex="-1" class="tce-error-summary tce-border-4 tce-rounded tce-p-3 tce-mt-2 tce-mb-5">
+  <h2 class="tce-text-2xl"><strong>There is a problem</strong></h2>
```

Comment:
Think adding a ! might look good

Reply:
Yes that might be good. I'll put some screenshots with and without the icon in Teams and ask for some feedback.

Reply:
It looks like on balance going without the '!' was preferred.